tests/docker/test_runner/Dockerfile

```diff
@@ -33,34 +27,47 @@ RUN mkdir -p /etc/apt/keyrings && \
     echo "deb [signed-by=/etc/apt/keyrings/renci-irods-archive-keyring.pgp arch=amd64] https://packages.irods.org/apt/ $(lsb_release -sc) main" | \
         tee /etc/apt/sources.list.d/renci-irods.list
 
-#### Install icommands - used to set up, validate and tear down tests. ####
+#### install basic packages ####
 RUN apt-get update && \
-    apt-get install -y irods-icommands && \
+    apt-get install -y \
+        curl \
+        python3 \
+        python3-distro \
+        python3-jsonschema \
+        python3-pip \
+        python3-psutil \
+        python3-pyodbc \
+        python3-requests \
+        unzip \
+    && \
     rm -rf /tmp/*
 
-#### install basic packages ####
-RUN apt-get install -y curl \
-    unzip \
-    python3 \
-    python3-distro \
-    python3-psutil \
-    python3-jsonschema \
-    python3-requests \
-    python3-pip \
-    python3-pyodbc
-
-RUN pip3 install botocore boto3 minio
+RUN python3 -m pip install botocore boto3 minio
 
 # install AWS CLI #
 
-RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
-RUN unzip awscliv2.zip
-RUN ./aws/install
+RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip" && \
+    unzip awscliv2.zip && \
+    ./aws/install
 
 # install MinIO client (mc) #
 
-RUN wget https://dl.min.io/client/mc/release/linux-amd64/mc
-RUN chmod +x mc
-RUN mv mc /usr/bin
+RUN wget https://dl.min.io/client/mc/release/linux-amd64/mc && \
+    chmod +x mc && \
+    mv mc /usr/bin
```

Comment:
Is moving binaries into /usr/bin what minio instructs people to do?

Reply:
Yeah, that's what I'd expect. Let's see if we can find another way to make `mc` available. Moving it into /usr/bin like that feels wrong and could potentially cause problems later on (I don't really expect this will be an issue, but you never know).

Reply:
Let's drop the use of $HOME and let /minio_binaries be the directory. That protects us from future surprises within container land.

Reply:
Done